src/Controller/NphOrderController.php

```diff
@@ -258,7 +258,7 @@ public function sampleFinalizeAction(
                     $sampleFinalizeForm['aliquotError']->addError(new FormError('Please enter at least one aliquot'));
                 } elseif ($nphOrderService->hasDuplicateAliquotsInForm($formData, $sampleCode)) {
                     $sampleFinalizeForm['aliquotError']->addError(new FormError('Please enter a unique aliquot barcode'));
-                } elseif ($duplicate = $nphOrderService->checkDuplicateAliquotId($formData, $sampleCode)) {
+                } elseif ($duplicate = $nphOrderService->checkDuplicateAliquotId($formData, $sampleCode, $sample->getNphAliquotIds())) {
```

Comment:
Could you write this without defining the variable in the `elseif()` statement?  The code here is already deeply nested, which would be nice to clean up at some point, but it would be good to avoid nesting a variable definition inside of control structure.